packages/combobox/src/Combobox/Combobox.helpers.ts

```diff
@@ -0,0 +1,12 @@
+export type Option = { value: string; label?: string };
+export const isMultiSelect = (
+  value: Option | Array<Option>,
+): value is Array<Option> => {
+  return Array.isArray(value);
+};
+
+export const isSingleSelect = (
+  value: Option | Array<Option>,
+): value is Option => {
+  return typeof value === "object" && !Array.isArray(value);
+};
```

Comment:
For narrowing types in the Combobox so we can properly type instances of Single Select & Multi Select